supervisor/docker/__init__.py

```diff
@@ -56,10 +56,10 @@ class DockerInfo:
     def new(data: dict[str, Any]):
         """Create a object from docker info."""
         return DockerInfo(
-            AwesomeVersion(data["ServerVersion"]),
-            data["Driver"],
-            data["LoggingDriver"],
-            data["CgroupVersion"],
+            AwesomeVersion(data.get("ServerVersion", "0.0.0")),
+            data.get("Driver", "unknown"),
+            data.get("LoggingDriver", "unknown"),
+            data.get("CgroupVersion", "1"),
```

Comment:
1 is what we want for cgroup version right?
Should we make this "0" to indicate a problem?

Reply:
The problem is: It doesn't have to be a problem?

Reply:
If its not a problem, this PR does not make sense

Reply:
in case of cgroup, no cgroup means v1

Reply:
the others can change on docker updates, at least we would start and not break

Reply:
btw, in future we support cgroup2 as well

Reply:
There are 2 things:

1. Docker doesn't provide the info
2. The cgroup version wrong

This mitigates 1 (preventing it from crashing), but it also means we don't know anything about 2. We can't tell if it is wrong or right